### PR TITLE
Finished refactoring.

### PR DESCRIPTION
**Pull request by blinker1104**
I think I forgot to update package.json.
I will update asap.

### CODE REVIEW

File: client/app.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    };
+    // this.getBizInfo = this.getBizInfo.bind(this);
+  }
+
+  componentDidMount() {
+    // this.getBizInfo.call(this);
+    this.getBizInfo();
+  }
+
+  getBizInfo() {
```
Comment (nathanksou) [inline client/app.jsx:29]: Is this function being used? I don't see it being passed down.

File: client/app.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      bizInfo: {},
+      loadComplete: false
+    };
+    // this.getBizInfo = this.getBizInfo.bind(this);
```
Comment (nathanksou) [inline client/app.jsx:21]: Probably unnecessary code here since you're calling it in scope.

Comment (blinker1104) [reply client/app.jsx:21]: I just put it as a placeholder tbh.
I will fix it and merge after.

Thanks

**Comment by blinker1104** [comment]
> Changes are good. I would note that in some components you can make them stateless functional components so there's less bloat in your code.

Modify some component stateless

**Pull request merged**